examples/bq582kh2487.json

```diff
@@ -0,0 +1,39 @@
+{
+  "@context": "http://sdr.sul.stanford.edu/contexts/taco-base.jsonld",
+  "@type": "http://sdr.sul.stanford.edu/models/sdr3-object.jsonld",
+  "citation": "Schwend, Rudolf, Mayor of Zurich, ?-1391. https://purl.stanford.edu/bq582kh2487",
+  "currentVersion": true,
```

Comment:
I'm still not sure I understand the usage for this.  Would anyone ever use `false`? When?